templates/taxi/car_detail.html

```diff
@@ -9,4 +9,6 @@
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <a class="btn btn-primary" href="{% url 'taxi:car-update' car.id %}">Update</a>
+  <a class="btn btn-danger" href="{% url 'taxi:car-delete' car.id %}">Delete</a>
```

Comment:
Similarly, change `car.id` to `car.pk` in the URL tag for consistency and to follow Django's convention for primary keys.